packages/frontend/stalker-app/src/app/modules/hosts/list-hosts/list-hosts.component.html

```diff
@@ -4,17 +4,28 @@
       <mat-card-title class="tw-flex-1" i18n="Hosts|Hosts">Hosts</mat-card-title>
 
       <div class="tw-flex tw-gap-sm tw-pb-4">
-        <button mat-stroked-button (click)="blockBatch(selection.selected)">
+        <button mat-stroked-button (click)="blockBatch(selection.selected)" [hasScopes]="['resources:hosts:update']">
           <mat-icon>block</mat-icon>
           <label i18n="Block|Block an item">Block</label>
         </button>
 
-        <button mat-stroked-button (click)="openNewHostsDialog(addHostsDialog)">
+        <button
+          mat-stroked-button
+          (click)="openNewHostsDialog(addHostsDialog)"
+          [hasScopes]="['resources:hosts:create', 'manage:projects:read']"
+          [requiresAllScopes]="true"
```

Comment:
I wonder if having two separate directives, `[hasScopes.anyOf]` and `[hasScopes.allOf]` would be clearer. At first I was like why are any of those scopes required, and then I saw the directive, but it wasnt obvious at first glance

We can also ignore this and merge